imageroot/systemd/controller-api.service

```
@@ -16,6 +16,7 @@ ExecStart=/usr/bin/podman run \
     --cgroups=no-conmon \
     --pod-id-file %t/MODULE_ID.pod-id \
     --replace -d --name MODULE_ID-api \
+    -v MODULE_ID-credentials:/nethsecurity-api/credentials/:z \
```

Other switches are in long format:
```suggestion
    --volume MODULE_ID-credentials:/nethsecurity-api/credentials/:z \
```